be/src/http/action/update_config_action.cpp

```diff
@@ -165,6 +165,10 @@ Status UpdateConfigAction::update_config(const std::string& name, const std::str
             (void)thread_pool->update_max_threads(
                     std::max(MIN_TRANSACTION_PUBLISH_WORKER_COUNT, config::transaction_publish_version_worker_count));
         });
+        _config_callback.emplace("transaction_publish_version_thread_pool_num_min", [&]() {
+            auto thread_pool = ExecEnv::GetInstance()->agent_server()->get_thread_pool(TTaskType::PUBLISH_VERSION);
+            (void)thread_pool->update_min_threads(config::transaction_publish_version_thread_pool_num_min);
```

Comment:
should check `MIN_TRANSACTION_PUBLISH_WORKER_COUNT`

Reply:
thks,review. I had added the check.